ale_linters/haskell/ghc.vim

```diff
@@ -16,3 +16,12 @@ call ale#linter#Define('haskell', {
 \   'command': 'stack ghc -- -fno-code -v0 %t',
 \   'callback': 'ale#handlers#haskell#HandleGHCFormat',
 \})
+
+call ale#linter#Define('haskell', {
```

Comment:
Could you move this to a separate file? The other two should probably also be in their own files.